nucypher/network/nodes.py

```diff
@@ -827,11 +835,19 @@ def learn_from_teacher_node(self, eager=False, canceller=None):
             return RELAX
 
         try:
-            response = self.network_middleware.get_nodes_via_rest(
-                node=current_teacher,
-                announce_nodes=announce_nodes,
-                fleet_state_checksum=self.known_nodes.checksum,
+            optional_latency_context_manager = (
+                self.node_latency_collector.get_latency_tracker(
+                    current_teacher.checksum_address
+                )
+                if self.node_latency_collector
+                else contextlib.nullcontext()
```

Comment:
This basically just makes the context manager... do nothing... in case of it being switched off, correct?  What's your motivation for installing a feature switch?  Do you plan to expose it via ursula config or CLI?

Reply:
> This basically just makes the context manager... do nothing... in case of it being switched off, correct?

Yep. 

> What's your motivation for installing a feature switch? Do you plan to expose it via ursula config or CLI?

Related to this - https://github.com/nucypher/nucypher/pull/3562#discussion_r1830745901.

Ursula has no *current* use for the statistics, but Bob and Porter do.

Reply:
Ah right, makes sense. 👍🏻

Reply:
I linked to the wrong comment above - updated.